src/Lykke.Service.ClientSearch/Modules/ServiceModule.cs

```diff
@@ -40,35 +40,34 @@ protected override void Load(ContainerBuilder builder)
         {
             builder.RegisterInstance(_log)
                 .As<ILog>()
-                .SingleInstance();
-
+                .SingleInstance();
+
             builder.RegisterType<HealthService>()
                 .As<IHealthService>()
                 .SingleInstance();
 
             builder.RegisterType<StartupManager>()
-                .As<IStartupManager>();
+                .As<IStartupManager>()
+                .SingleInstance();
 
             builder.RegisterType<ShutdownManager>()
-                .As<IShutdownManager>();
-
-
-
-            builder.RegisterInstance<ITriggerManager>(new TriggerManager(new AutofacServiceProvider(_applicationContainer)));
-            builder.RegisterInstance<IPersonalDataService>(new PersonalDataService(_pdClientSettings.CurrentValue, _log));
-
+                .As<IShutdownManager>();
+
+
+            builder.RegisterInstance<IPersonalDataService>(new PersonalDataService(_pdClientSettings.CurrentValue, _log));
```

Comment:
PersonalDataService can be registered as builder.RegisterType< PersonalDataService >().As< IPersonalDataService >()

Reply:
rewritten